scripts/zones/Apollyon/mobs/Inhumer.lua

```diff
@@ -2,23 +2,105 @@
 -- Area: Apollyon SE
 --  Mob: Inhumer
 -----------------------------------
-require("scripts/globals/limbus");
------------------------------------
+require("scripts/globals/pathfind")
+local ID = require("scripts/zones/Apollyon/IDs")
+local flags = tpz.path.flag.WALLHACK
+local path =
+{
+    [1] = 
+    {
+        {391.345, 0.000, -362.357},
+        {409.661, -0.500, -372.819}
+    },
+    [2] = 
+    {
+        {334.872, 0.000, -336.313},
+        {341.842, -0.764, -319.744}
+    },
+    [3] = 
+    {
+        {362.613, 0.000, -276.235},
+        {344.407, -1.398, -280.625}
+    },
+    [4] = 
+    {
+        {327.796, 0.000, -320.106},
+        {336.038, -1.286, -310.834}
+    },
+    [5] = 
+    {
+        {326.597, 0.000, -313.956},
+        {322.230, -0.493, -328.462}
+    },
+    [6] = 
+    {
+        {363.584, 0.000, -300.336},
+        {348.563, -1.110, -312.928}
+    },
+    [7] = 
+    {
+        {331.868, 0.000, -337.062},
+        {330.350, -0.481, -323.224}
+    },
+    [8] = 
+    {
+        {340.493, -0.879, -309.115},
+        {339.728, -0.582, -320.195}
+    }
+}
 
-function onMobDeath(mob, player, isKiller)
-end;
+function onMobRoam(mob)
+    local offset = mob:getID() - ID.mob.APOLLYON_SE_MOB[3]
+    local pause = mob:getLocalVar("pause")
+    local start = mob:getLocalVar("start") == 1
+    if pause < os.time() and start then
+        local point = (mob:getLocalVar("point") % 2)+1
+        mob:setLocalVar("point", point)
+        mob:pathTo(path[offset][point][1], path[offset][point][2], path[offset][point][3], flags)
+        mob:setLocalVar("pause", os.time()+30)
+    end
+end
 
-function onMobDespawn(mob)
- local mobID = mob:getID();
- if (mobID ==16933025) then -- time
-    GetNPCByID(16932864+246):setPos(343,-1,-296);
-    GetNPCByID(16932864+246):setStatus(tpz.status.NORMAL);
- elseif (mobID ==16933028) then -- recover
-    GetNPCByID(16932864+248):setPos(376,-1,-259);
-    GetNPCByID(16932864+248):setStatus(tpz.status.NORMAL);
- elseif (mobID ==16933022) then -- item
-    GetNPCByID(16932864+247):setPos(307,-1,-309);
-    GetNPCByID(16932864+247):setStatus(tpz.status.NORMAL);
- end
+function onMobSpawn(mob)
+    mob:setMod(tpz.mod.HTHRES, 1500)
+    mob:setMod(tpz.mod.IMPACTRES, 1500)
+    mob:setMod(tpz.mod.PIERCERES, 0)
+end
 
-end;
+function onMobEngaged(mob, target)
+    local start = mob:getLocalVar("start") == 1
+    if not start then
+        for i = 1, 8 do
+            GetMobByID(ID.mob.APOLLYON_SE_MOB[3]+i):setLocalVar("pause", os.time()+(3 * i))
+            GetMobByID(ID.mob.APOLLYON_SE_MOB[3]+i):setLocalVar("start", 1)
```

Comment:
What's going on here during engage? Are we trying to get the others to stop pathing?

Reply:
I may be wrong here, but if you're careful when you spawn into this floor to not get aggro by the one close to where you spawn in, it doesn't look like they start roaming until one of them engages. May need further confirmation. That's what it looked like to me and made sense as some "wake the dead" mechanic.